2022-05-metafacture-workshop/index.html

```diff
@@ -0,0 +1,939 @@
+						Maintainer</span></p>
+				<p><span class="fragment"><a href="https://github.com/metafacture/metafacture-fix/graphs/contributors">2019: Start von Metafacture Fix</a></span></p>
+				<p><span class="fragment"><a href="https://github.com/metafacture/metafacture-playground/graphs/contributors">2021: Start von Metafacture Playground</a></span></p>
+			</section>
+
+			<section>
+				<h2>Wie Metafacture funktioniert</h2>
+				<p><span class="fragment">Grundidee: Daten fließen durch mehrere Module: <br />&rarr; read &rarr; decode
+						&rarr; transform &rarr; encode &rarr; write &rarr;</span></p>
+				<p><span class="fragment">Jedes Modul erwartet Input eines bestimmten Typs und erzeugt Output eines
+						bestimmten Typs</span></p>
+				<p><span class="fragment">Verschiedene Formate werden unterstützt (z.B. PICA, MARC), erweiterbares
+						Framework für eigene Formate</span></p>
+				<p><span class="fragment">Durch Kombination einzelner Module, durch die unsere Daten fließen, bauen wir
+						den Workflow</span></p>
+			</section>
+
+			<section>
+				<h1>Ein Workflow</h1>
+				<img src="img/flow.png" />
+				<p><small>Aus: Christoph Böhme, <a
+							href="http://swib.org/swib13/slides/boehme_swib13_131.pdf">http://swib.org/swib13/slides/boehme_swib13_131.pdf</a></small>
+				</p>
+			</section>
+
+			<section>
+				<h2>Workflows konfigurieren und ausführen</h2>
+				<p><span class="fragment">Workflows können in Flux (einer speziellen Konfigurationssprache) oder mit
+						Java (typsicher über Java Generics) bearbeitet werden</span></p>
+				<p><span class="fragment">Flux-Workflows können in einem Texteditor editiert und auf der Kommandozeile
+						ausgeführt werden, Java-Workflows funktionieren wie andere Java-Komponenten</span></p>
+			</section>
+
+			<section>
+				<h2>Metafacture Playground</h2>
+				<p><span class="fragment">Webbasierte Oberfläche zum Ausprobieren und Austauschen von Workflows</span></p>
+				<p><span class="fragment">Ziel: Einstiegshürde für Metafacture senken, neben Morph-Komplexität unserer Erfahrung nach ein weiteres zentrales Problem bei der Metafacture-Nutzung</span></p>
+				<p><span class="fragment">Für Entwicklung, Dokumentation, Tutorials, Workshops</span></p>
+				<p><span class="fragment"><a
+							href="https://metafacture.org/playground">https://metafacture.org/playground</a></span></p>
+			</section>
+
+			<section>
+				<a href="https://metafacture.org/playground/?flux=PG_DATA%0A%7Cas-lines%0A%7Cdecode-formeta%0A%7Cfix%0A%7Cencode-json%0A%7Cprint%0A%3B&fix=move_field%28_id%2C+id%29%0Amove_field%28a%2C+title%29%0Apaste%28author%2C+b.v%2C+b.n%2C+%27~aus%27%2C+c%29%0Aretain%28id%2C+title%2C+author%29&data=1%7Ba%3A+Faust%2C+b+%7Bn%3A+Goethe%2C+v%3A+JW%7D%2C+c%3A+Weimar%7D%0A2%7Ba%3A+R%C3%A4uber%2C+b+%7Bn%3A+Schiller%2C+v%3A+F%7D%2C+c%3A+Weimar%7D&active-editor=fix"><img src="img/playground.png" style="border: none;" /></a>
+			</section>
+
+			<section>
```

Comment:
```suggestion
			<section>
			    Wie funktioniert das in der Praxis? Lasst uns das gemeinsam ausprobieren.
			</section>
			<section>
```

Folie einschieben, dass man jetzt praktisch etwas macht.